c_width: 1280
c_height: 800
c_fps: 60

c_k1: -0.389835
c_k2: 0.161253
c_p1: -0.002684
c_p2: 0.002155

c_fx: 1015.820924
c_fy: 1016.564409
c_cx: 639.5000000
c_cy: 399.5000000
